fabric-rendering-v1/src/main/resources/fabric-rendering-v1.mixins.json

```diff
@@ -8,7 +8,12 @@
     "MixinBuiltinModelItemRenderer",
     "MixinInGameHud",
     "MixinItemColorMap",
-    "MixinWorldRenderer"
+    "MixinWorldRenderer",
+    "EntityModelLayersAccessor",
+    "EntityModelsMixin",
+    "LivingEntityRendererAccessor",
+    "MixinBlockEntityRenderers",
+    "MixinEntityRenderers"
```

Comment:
Preferably this would be sorted alphabetically like in other modules, but this is fine too.